src/components/Chart/resolveLabelsOnBars.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-underscore-dangle */
+import ChartJS from 'chart.js';
+import defaultTheme from '../../styles/defaultTheme';
+
+const palette = defaultTheme.palette;
```

Comment:
You can't just use the defaultTheme, I think a possible solution is to pass the parameter palette to this function when you call it in resolveOptions.js

Reply:
I tried that at first but the function loses the instance of the chart when I pass some parameter, however check my next commit, I made a change regarding this, anyway tell me if you see a better solution